CHANGELOG.md

```diff
@@ -26,18 +36,16 @@ This file is used to list changes made in each version of the splunk cookbook.
 - enhancements to the `:remove` action for `splunk_installer` resource to ensure a complete uninstall for both Splunk Enterprise Server and Universal Forwarder
 
 ## 6.4.1 (2020-10-20)
+
 - Fixes an issue running Splunk and the Splunk Universal Forwarder as a non-root user
 - Fixes Test Kitchen configuration to test running Splunk as non-root user
 - Helper methods in `libraries/helper.rb` are moved to their own module space: `ChefSplunk::Helpers`
 - Disables Splunk management port (8089) when installing the Universal Forwarder
 
 ## 6.4.0 (2020-10-19)
+
 - Fixes Issue [#185](https://github.com/chef-cookbooks/chef-splunk/issues/185)
-  * a startup issue was resolved for SplunkForwarder installations with an improved
-    systemd unit file (fix below)
-  * Adds Inspec tests to verify from SplunkForwarder starts (thanks, [@jjm](https://github.com/jjm))
```

Comment:
Why did you remove the content above?

Reply:
I thought sub-lists werent allowed, Ill revert that

Reply:
I don't believe so, as long as they are formatted correctly. You can check the Markdown spec though.